packages/yew/src/html/conversion.rs

```diff
@@ -0,0 +1,89 @@
+use super::{Component, NodeRef, Scope};
+use std::{borrow::Cow, rc::Rc};
+
+/// Marker trait for types that the [`html!`] macro may clone implicitly.
+pub trait ImplicitClone: Clone {}
+
+// this is only implemented because there's no way to avoid cloning this value
+impl ImplicitClone for Cow<'static, str> {}
+
+impl<T: ImplicitClone> ImplicitClone for Option<T> {}
+impl<T> ImplicitClone for Rc<T> {}
+
+impl ImplicitClone for NodeRef {}
+impl<Comp: Component> ImplicitClone for Scope<Comp> {}
+// TODO there are still a few missing like AgentScope
```

Comment:
I did the AgentScope... Can someone help me enumerate what else is missing? :sweat_smile:

Reply:
I can't think of anything else that is missing. In the worst case, the user will have to call `clone` if this trait isn't implemented. Let's just roll with it for now. We'll know what's missing once this code is merged and used.

Reply:
Fine by me